index.js

```diff
@@ -43,7 +44,14 @@ const Directors = Models.Directors;
 //local
 // mongoose.connect('mongodb://localhost:27017/movieMongoDB', { useNewUrlParser: true, useUnifiedTopology: true });
 //heroku
-mongoose.connect( process.env.CONNECTION_URI);
+mongoose.connect(process.env.CONNECTION_URI)
```

Comment:
connect returns a promise, I am resolving the promise so you can tell when the mongoose connect fail or succeeds